app/controllers/packages_controller.rb

```diff
@@ -61,6 +81,27 @@ def packages
     Package.configure(config)
   end
 
+  # Smooth over slicing out only the info RM API needs to create a
+  # custom package This also passes along the required provider id
+  def rm_api_package
+    find_provider
+
+    relevent_rm_api_data = package_params.slice(
```

Comment:
relevant*.  also maybe just drop that part entirely from the variable name, imo.  it's relevant by virtue of the fact that we're doing something with it.